packages/core/components/Drawer/styles.module.css

```diff
@@ -2,6 +2,12 @@
   font-family: var(--puck-font-family);
 }
 
+.DrawerItem--disabled {
```

Comment:
Just realised, this should be applied to the `-draggable`. I'll mark this as a follow-up.

Reply:
Follow-up: https://github.com/measuredco/puck/issues/580